### PR TITLE
Add correct CCLA and ICLA [skip ci]

### PR DESCRIPTION
**Pull request by jmertic**

### CODE REVIEW

**Comment by danrbailey** [comment]
Hi John, is it necessary to introduce these CLA forms? Are we not able to suggest that people only fill out the digital CLA through EasyCLA now?

(PS if you push a new empty commit, I expect the failed check will succeed, some issue with the Window builder here)

**Comment by jmertic** [comment]
Hey @danrbailey! It's certainly not required, but we've seen several of the other projects maintain the CLA templates in GitHub for contributors to reference ( EasyCLA doesn't make it too terribly visible :-( ). Also you will run into the occasional company that won't use EasyCLA, so having it in markdown format ( which is a pandoc away from PDF or DOC format ) solves that issue.

Lemme know if this seems reasonable - happy either way...

**Comment by danrbailey** [comment]
> Hey @danrbailey! It's certainly not required, but we've seen several of the other projects maintain the CLA templates in GitHub for contributors to reference ( EasyCLA doesn't make it too terribly visible :-( ). Also you will run into the occasional company that won't use EasyCLA, so having it in markdown format ( which is a pandoc away from PDF or DOC format ) solves that issue.
> 
> Lemme know if this seems reasonable - happy either way...

Perhaps we could put a note that effect at the top of these CLAs to try and avoid any confusion? Ie state that EasyCLA is our preferred mechanism for signing CLAs.

**Comment by jmertic** [comment]
Hey @danrbailey! I'd prefer to avoid adding additional text to the CLAs as they are legal documents. However, I did update the verbiage in the README and CONTRIBUTING file to spell things out more clearly and hopefully clear up confusion folks might have. Let me know if that's a reasonable approach.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 ![OpenVDB](https://www.openvdb.org/images/openvdb_logo.png)
 
-[![Azure](https://dev.azure.com/academysoftwarefoundation/Academy%20Software%20Foundation/_apis/build/status/academysoftwarefoundation.openvdb)](https://dev.azure.com/academysoftwarefoundation/Academy%20Software%20Foundation/_build?definitionId=1&_a=summary)
+[![License](https://img.shields.io/badge/License-BSD%203%20Clause-blue.svg)](LICENSE.md)
```
Comment (danrbailey) [inline README.md]: Think this should be Mozilla 2.0?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,62 @@
 # Overview
 
-This project aims to be governed in a transparent, accessible way for the benefit of the community. All participation in this project is open and not bound to corporate affilation. Participants all our bound to the ASWF Code of Conduct.
+This project aims to be governed in a transparent, accessible way for the benefit of the community. All participation in this project is open and not bound to corporate affilation. Participants all our bound to the [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (danrbailey) [inline CONTRIBUTING.md]: affilation => affiliation

Comment (danrbailey) [reply CONTRIBUTING.md]: Participants all our bound => Participants are all bound

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,19 +1,62 @@
...
+in OpenVDB must first complete a **Contributor
+License Agreement**.
+
+OpenVDB uses [EasyCLA](https://lfcla.com/) for managing CLAs, which automatically
+checks to ensure CLAs are signed by a contributor before a commit
+can be merged. 
+
+* If you are an individual writing the code on your own time and
+  you're SURE you are the sole owner of any intellectual property you
+  contribute, you can [signing the CLA as an individual contributor](https://github.com/communitybridge/easycla/blob/master/docs/sign-a-cla-as-an-individual-contributor-to-github.md).
```
Comment (danrbailey) [inline CONTRIBUTING.md]: signing => sign

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,19 +1,62 @@
...
+
+* If you are writing the code as part of your job, or if there is any
+  possibility that your employers might think they own any
+  intellectual property you create, then you should use the [Corporate
+  Contributor Licence
+  Agreement](https://github.com/communitybridge/easycla/blob/master/docs/contribute-to-a-github-company-project.md).
+
+The OpenVDB CLA's are the standard forms used by Linux Foundation
```
Comment (danrbailey) [inline CONTRIBUTING.md]: CLA's => CLAs

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,19 +1,62 @@
...
+projects and [recommended by the ASWF TAC](https://github.com/AcademySoftwareFoundation/tac/blob/master/process/contributing.md#contributor-license-agreement-cla). You can review the text of the CLAs in the [TSC directory](tsc/).
+
+#### Commit Sign-Off
+
+Every commit must be signed off.  That is, every commit log message
+must include a “`Signed-off-by`” line (generated, for example, with
+“`git commit --signoff`”), indicating that the committer wrote the
+code and has the right to release it under the
+[Modified-BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause)
```
Comment (danrbailey) [inline CONTRIBUTING.md]: Should be Mozilla 2.0 license

File: README.md (hunk 1 above)
Comment (jmertic) [reply README.md]: Fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (jmertic) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (jmertic) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 4 above)
Comment (jmertic) [reply CONTRIBUTING.md]: Fixed

File: CONTRIBUTING.md (hunk 5 above)
Comment (jmertic) [reply CONTRIBUTING.md]: Fixed

**Pull request merged**